andela/ah-titans-frontend
# #160415392 Create View an Article

#### What does this PR do?
Allows users to view an article they have created or a specific article

#### Description of tasks to be completed
1. Add loader when fetching an article
2. Display an article that has been created

#### How should this be manually tested?
```sh
$ git clone https://github.com/andela/ah-titans-frontend.git
```
```sh
$ cd ah-titans-frontend
```
```sh
$ git checkout -b ft-users-able-to-view-article-160415392
```
```sh
$ npm install
```
```sh
$ npm start
```
>Sign up/Sign in to the app
>Click the floating add icon at the bottom of the home page and create an article

<img width="1234" alt="screen shot 2018-09-18 at 12 44 07" src="https://user-images.githubusercontent.com/29487740/45679111-9ceed900-bb40-11e8-913f-fb08adf5557f.png">

<img width="1219" alt="screen shot 2018-09-18 at 12 42 54" src="https://user-images.githubusercontent.com/29487740/45679137-aaa45e80-bb40-11e8-8d5f-6d70d4854276.png">

>Wait for the page to load as the article is being fetched

<img width="1280" alt="screen shot 2018-09-18 at 12 43 11" src="https://user-images.githubusercontent.com/29487740/45679174-be4fc500-bb40-11e8-8f6c-5ef967d1de04.png">

>You should be redirected to the article you have created

<img width="1231" alt="screen shot 2018-09-18 at 12 43 28" src="https://user-images.githubusercontent.com/29487740/45679180-c1e34c00-bb40-11e8-824b-2f9816abdfca.png">

#### Any background context you want to provide?
No

#### What are the relevant Pivotal Tracker Stories?
[#160415392](https://www.pivotaltracker.com/story/show/160415392)
